Packs/Code42/IncidentFields/incidentfield-Code42_SecurityAlert_Description.json

```diff
@@ -19,7 +19,7 @@
   "isReadOnly": false,
   "locked": false,
   "mergeStrategy": "",
-  "name": "Code42 Alert Description",
+  "name": "Code42 SecurityAlert Description",
```

Comment:
Weird, but the validate command got mad for using the word Alert.

Reply:
I'm guessing that XSOAR has a concept of alerts that they want to make sure nothing collides with.